#! /bin/sh /usr/share/dpatch/dpatch-run
## 82-fix-PSS-INSBIN.dpatch by Thomas <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: remove extra INSBIN that causes psdl_tao to be copied in the toplevel
## DP: directory.
--- ACE_wrappers.orig/TAO/orbsvc/PSS/GNUmakefile.PSDL_Parser	2005-03-16 21:24:26.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/PSS/GNUmakefile.PSDL_Parser	2005-03-16 21:25:16.000000000 +0100
@@ -28,7 +28,6 @@
 TAO_IDL_DEP = ../../../bin/tao_idl$(EXEEXT)
 TAO_IDLFLAGS = -Ge 1 -Wb,pre_include=ace/pre.h -Wb,post_include=ace/post.h -I../..
 
-INSBIN ?= .
 ifeq ($(INSBIN),.)
   ifeq ($(PWD),)
     PWD=$(shell pwd)
